extension/content_script.js

```diff
@@ -183,19 +179,10 @@ function moveOne(dir) { // Sets start and end
 
 // Scroll up when tracker is above page
 function scrollUp() {
-	// Check to see if we should use old style or look for new
-	let tracker_style_current = null; // Depends on if style was just switched
-	if ($("."+trackerStyle)) {
-		tracker_style_current = trackerStyle
-	} else { 
-		let displaySettings = display.getSettings();
-		tracker_style_current = "trackerHighlighter"+displaySettings[1]+"Shadow"+displaySettings[2]; 
-	}
-
```

Comment:
No longer need to check for this; just need to look for "trackerClass", which is static